client/payment-method-icons/bancontact/icon.svg

```diff
@@ -0,0 +1 @@
a1.674 1.674 0 0 1-.05-.448v-.664c0-.183.017-.332.05-.448a.893.893 0 0 1 .133-.282.631.631 0 0 1 .464-.2c.233 0 .415 0 .548.017.133.017.282.017.431.034h.017c.05 0 .083-.017.116-.034.033-.016.05-.05.05-.1v-.713c0-.066-.017-.1-.033-.116-.034-.016-.067-.05-.133-.066-.116-.033-.249-.05-.431-.083-.183-.033-.382-.033-.631-.033-.564 0-1.012.166-1.36.514-.332.349-.515.846-.515 1.51v.664c0 .647.166 1.161.514 1.51.332.348.797.514 1.361.514.232 0 .448-.017.63-.033.183-.033.316-.05.432-.083a.25.25 0 0 0 .133-.067c.033-.016.033-.066.033-.116Zm-3.916.066v-2.655c0-.249-.017-.481-.066-.714a1.62 1.62 0 0 0-.216-.58.966.966 0 0 0-.431-.382c-.183-.1-.415-.15-.714-.15a2.17 2.17 0 0 0-.68.1 2.33 2.33 0 0 0-.63.332v-.199a.15.15 0 0 0-.05-.116.15.15 0 0 0-.117-.05H9.89a.15.15 0 0 0-.116.05.15.15 0 0 0-.05.116v4.248c0 .05.017.083.05.116a.15.15 0 0 0 .116.05h.963a.15.15 0 0 0 .116-.05.15.15 0 0 0 .05-.116v-3.136c.132-.067.265-.133.398-.183a.945.945 0 0 1 .365-.066c.116 0 .215.016.298.033.083.017.133.066.183.116.05.067.066.133.083.233.017.099.017.215.017.348v2.655c0 .05.016.083.05.116a.15.15 0 0 0 .115.05h.963a.15.15 0 0 0 .116-.05c.05-.033.083-.083.083-.116Zm-6.173-.913c-.1.05-.199.083-.315.116-.116.034-.232.05-.348.05-.166 0-.299-.017-.382-.066-.083-.05-.116-.15-.116-.332v-.067c0-.1.017-.182.033-.248a.43.43 0 0 1 .116-.183c.05-.05.133-.083.216-.116.083-.017.2-.033.349-.033h.447v.88Zm1.278-1.941c0-.299-.05-.548-.133-.747a1.386 1.386 0 0 0-.365-.498 1.359 1.359 0 0 0-.58-.282 2.864 2.864 0 0 0-.78-.1c-.266 0-.531.017-.78.05a3.456 3.456 0 0 0-.597.117c-.1.033-.15.083-.15.182v.647c0 .05.017.083.033.116.034.017.067.034.1.034h.033c.067 0 .15-.017.232-.017.1 0 .2-.017.332-.017.116 0 .25-.016.382-.016h.382c.182 0 .331.033.431.1.1.066.166.215.166.447v.282h-.431c-.68 0-1.195.1-1.494.316-.299.216-.464.564-.464 1.029v.066c0 .265.033.481.116.647.083.183.182.315.315.431.133.1.265.183.431.233.166.05.332.066.515.066a1.7 1.7 0 0 0 .614-.1 2.42 2.42 0 0 0 .497-.265v.133c0 .05.017.083.05.116a.15.15 0 0 0 .116.05h.896a.15.15 0 0 0 .117-.05.15.15 0 0 0 .05-.116v-2.854h-.034Zm-5.708 1.26a.632.632 0 0 1-.232.515c-.15.116-.448.183-.846.183h-.664v-1.494h.896c.315 0 .548.067.664.216a.8.8 0 0 1 .182.498v.083Zm-.033-2.372c0 .083-.017.166-.033.249a.423.423 0 0 1-.133.2c-.066.049-.15.099-.249.132-.1.033-.232.05-.398.05h-.896v-1.378h.614c.415 0 .697.05.863.15.166.1.249.265.249.481v.116h-.017Zm1.36 2.29c0-.282-.066-.514-.198-.714a1.261 1.261 0 0 0-.515-.464c.216-.1.382-.266.498-.465.116-.216.182-.448.182-.697v-.149c0-.315-.066-.58-.182-.797a1.39 1.39 0 0 0-.514-.514c-.216-.133-.482-.216-.797-.282a6.456 6.456 0 0 0-1.029-.083H1.46c-.133 0-.265.017-.398.017s-.249.016-.365.033c-.116.017-.2.017-.266.033-.149.033-.265.066-.331.15-.067.066-.1.198-.1.381v4.845c0 .183.033.299.1.382.066.066.182.116.331.15.083.016.183.032.282.032.117.017.233.017.366.034.132 0 .265.016.398.016h.415c.348 0 .68-.033.979-.083.298-.05.58-.15.796-.282.232-.133.398-.315.548-.548.132-.232.199-.53.199-.879v-.116Z" fill="#004E91"/></svg>
```

Comment:
These icons have all been exported from Figma and optimized via https://jakearchibald.github.io/svgomg/